yarn.lock

```diff
@@ -4772,17 +4772,6 @@
   resolved "https://registry.yarnpkg.com/@istanbuljs/schema/-/schema-0.1.3.tgz#e45e384e4b8ec16bce2fd903af78450f6bf7ec98"
   integrity sha512-ZXRY4jNvVgSVQ8DL3LTcakaAtXwTVUxE81hslsyD2AtoXW/wVob10HkOJ1X/pAlcI7D+2YoZKg5do8G/w6RYgA==
 
-"@janus-idp/shared-react@^1.3.0":
-  version "1.3.0"
-  resolved "https://registry.yarnpkg.com/@janus-idp/shared-react/-/shared-react-1.3.0.tgz#c945834ee77548ad0336f9f7b61b1e6d85cdcbe6"
-  integrity sha512-ulnSgcE/zyq/F0RYY+mZTY6kC7rug8BDJG82+xFowkhj6TtduvzIa63FBDD685pUXI5lPfqiAvxCVOnBTDx7tQ==
-  dependencies:
-    "@kubernetes/client-node" "^0.18.1"
-    classnames "2.3.2"
-    date-fns "^2.30.0"
-    lodash "^4.17.19"
-    mathjs "^11.9.1"
-
```

Comment:
Interesting.. Do we have `yarn.lock` out of sync?

Reply:
Do you want me to remove `yarn.lock`?

Reply:
Let's keep it if we're out of sync... 🤷

Reply:
@tumido

Reply:
@nickboldt we will need to build `shared-react` downstream since it is used in many of our plugins.

Reply:
ok, good, I've been doing that for a month now :D